tests/functional/c/consider/consider_refactoring_into_while_condition_py.txt

```diff
@@ -0,0 +1,22 @@
+consider-refactoring-into-while-condition:7:8:10:18:Issue8015.bad:"Consider using 'while k != 10' instead of 'while True:' an 'if', and a 'break'":HIGH
+consider-refactoring-into-while-condition:14:8:17:32:Issue8015.another_bad:"Consider using 'while current_scope is not None' instead of 'while True:' an 'if', and a 'break'":HIGH
+consider-refactoring-into-while-condition:27:8:32:18:Issue8015.test_error_message_multiple_break:"Consider using 'while k > 1' instead of 'while True:' an 'if', and a 'break'":HIGH
```

Comment:
Should we detect multiple breaks here and suggest this instead ?

```suggestion
consider-refactoring-into-while-condition:27:8:32:18:Issue8015.test_error_message_multiple_break:"Consider using 'while k > 1 and k <= 10' instead of 'while True:' an 'if', and a 'break'":HIGH
```

Reply:
implemented this